### PR TITLE
DOC changed documentation for axvspan to numpydoc format @NelleV

### PR DESCRIPTION
**Pull request by Wiolcia**
Updated documentation of axvspan function to numpydoc format, as part of a project during the ASPP2016 Summer School

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -856,39 +856,50 @@ def axvspan(self, xmin, xmax, ymin=0, ymax=1, **kwargs):
...
 
-        Valid kwargs are :class:`~matplotlib.patches.Polygon`
-        properties:
+        Other Parameters
+        ----------------
+        **kwargs
+            Optional parameters are properties of the class
+            matplotlib.patches.Polygon.
 
-        %(Polygon)s
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: Why remove this?

**Comment by QuLogic** [comment]
This PR should probably be coordinated with #7037 to get consistent prose.

**Comment by NelleV** [comment]
Apart from my comments, this look great!

Thanks,
N

File: lib/matplotlib/axes/_axes.py (hunk 1 above)
Comment (tacaswell) [inline lib/matplotlib/axes/_axes.py]: Agree with @QuLogic this should probably stay

Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: I think it should be removed and moved to a place with clean and readable summaries. These make the documentation unreadable in an Ipython terminal. They also make the documentation much harder to maintain and modify. They are also not standard, and thus makes contributing to the documentation much harder.

Comment (tacaswell) [inline lib/matplotlib/axes/_axes.py]: If these sorts of tables are going to be removed, it should be done in a coordinated way ( @efiring is also on board with this) not piecemeal.

**Comment by NelleV** [comment]
I am closing this PR: I've created a new one with the additional changes needed to merge this in.
Thanks for the contribution!

**Pull request closed without merging**